packages/frameworks/src/frameworks.ts

```diff
@@ -2056,7 +2056,6 @@ export const frameworks = [
         value: 'dist',
       },
     },
-    dependency: '@sanity/cli',
```

Comment:
`@sanity/cli` isn't a dependency for Sanity 3+ any longer. Unsure if this comment still applies?

https://github.com/vercel/vercel/blob/9d4d4b6deb6294506016106f78e71f1984adcc7f/packages/frameworks/src/frameworks.ts#L12-L16

Several framework entries don't have this value or don't reference a cli

Reply:
cc @TooTallNate @EndangeredMassa